terraform/usa.gov.tf

```diff
@@ -27,6 +27,28 @@ resource "aws_route53_record" "usa_gov_analytics_usa_gov_aaaa" {
   }
 }
 
+resource "aws_route53_record" "usa_gov_standards_usa_gov_a" {
+  zone_id = "${aws_route53_zone.usa_gov_zone.zone_id}"
+  name = "standards.usa.gov."
+  type = "A"
+  alias {
+    name = "d98p4kkshmqgd.cloudfront.net."
+    zone_id = "Z2FDTNDATAQYW2"
```

Comment:
@datn I think we want to use `zone_id = "${aws_route53_zone.primary.zone_id}"` rather than hardcoding for all of the `aws_route53_record`s in this file, right?